python/py-tensorflow-tensorboard/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tensorflow-tensorboard
+version             0.4.0rc3
+platforms           darwin
+license             Apache-2
+maintainers         @emcrisostomo openmaintainer
+
+description         TensorFlow helps the tensors flow
+long_description    TensorBoard is a suite of web applications for inspecting \
+                    and understanding your TensorFlow runs and graphs.
+
+homepage            https://github.com/tensorflow/tensorboard
+master_sites        https://pypi.python.org/packages/d5/fb/80df4eb3234c41beeeb3122b1966effbf08608fb80d2fa74bb72f8da9cb3
```

Comment:
Does `pypi:/t/tensorflow-tensorboard` not work here? The files.pythonhosted.org redirector should usually work to find the appropriate hex string subdir.

Reply:
Nope, I tried to make it work but with no luck. I think it's related to its being a `whl`. Look at the attempt:

```--->  Fetching distfiles for py27-tensorflow-tensorboard
--->  Attempting to fetch tensorflow_tensorboard-0.4.0rc3-py2-none-any.whl from https://files.pythonhosted.org/packages/source/t/tensorflow-tensorboard
--->  Attempting to fetch tensorflow_tensorboard-0.4.0rc3-py2-none-any.whl from https://pypi.python.org/packages/source/t/tensorflow-tensorboard
--->  Attempting to fetch tensorflow_tensorboard-0.4.0rc3-py2-none-any.whl from http://lil.fr.distfiles.macports.org/py-tensorflow-tensorboard
... snip ...
Error: Failed to fetch py27-tensorflow-tensorboard: ... snip ...```